migrations/0015-exhibition-publication-and-attributed-image-validation.js

```diff
@@ -0,0 +1,32 @@
+module.exports = function(migration) {
+  const exhibitionPage = migration.editContentType('exhibitionPage');
+
+  exhibitionPage
```

Comment:
This will put the new field at the end of the ones that already exist. I suggest moving it to immediately after `primaryImageOfPage` to be more consistent with the blog post content type.